x-pack/platform/plugins/private/snapshot_restore/public/application/components/repository_form/type_settings/azure_settings.tsx

```diff
@@ -160,6 +173,7 @@ export const AzureSettings: React.FunctionComponent<Props> = ({
                 }}
                 data-test-subj="containerInput"
                 disabled={isManagedRepository}
+                id={containerId}
```

Comment:
@paulinashakirova  unfortunately it's not a good practice to set the same id for 2 or more DOM elements.